src/components/reservation/MyReservations.jsx

```diff
@@ -0,0 +1,175 @@
+import { useEffect, useState } from "react"; // Import useState
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchReservations,
+  deleteReservation,
+} from "../../features/reservation/reservSlice";
+import { fetchCars } from "../../features/cars/carSlice";
+import { toast } from "react-hot-toast";
+import { SwishSpinner } from "react-spinners-kit";
+import "../../assets/style/MyReservations.css";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Mousewheel, Pagination } from "swiper";
+import "swiper/css";
+import "swiper/css/pagination";
+
+function UserReservation() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const UserReservation= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/VelzckC0D3/Land-Rover-E-Commerce/assets/57421823/e2c09037-e76c-4f31-a595-8471e4713a4a)


That way, your code will meet the latest standards. 💪